src/onegov/feriennet/views/notification_template.py

```diff
@@ -230,8 +232,8 @@ def email_iter() -> 'Iterator[EmailJsonDict]':
         'title': _("Mailing"),
         'layout': layout,
         'form': form,
-        'preview_subject': variables.render(self.subject),
-        'preview_body': variables.render(self.text),
+        'preview_subject': variables.render(Markup(self.subject)),
```

Comment:
don't wrap the subject, the subject needs to be escaped.
```suggestion
        'preview_subject': variables.render(self.subject),
```